package.json

```diff
@@ -22,6 +22,7 @@
     "react-beautiful-dnd-grid": "^0.1.3-alpha",
     "react-dom": "^16.12.0",
     "react-ga": "^2.7.0",
```

Comment:
I would like to note here that react-grid-layout 0.17.0 is the only version of the library that does not have a place holder bug. Until they update it. Lets make sure everyone has this same version.